data_pipeline/all.json

```diff
@@ -0,0 +1,5138 @@
+{
```

Comment:
My general thought on this is that for this project "buildable" files like this probably shouldn't get committed. If someone changes a file on which this depends but doesn't run the `combine` script, then this file is now out of date. It also adds to the repo size unnecessarily.

I feel the better approach would be just to write the script, put the output file name in the `.gitignore`, then in those instructions you made just say like 'hey run the script and then upload the output file to firebase'.

Reply:
I believe the original reason to add it to the repo was to have some kind of version control on that data itself. We'll need to talk with @fredmon3 a bit more on that since we hadn't completely planned out how that would work.

Reply:
Yeah I mean that's fine.. have the two files that combine to make "all" under version control, just not the file that gets built with the two

Reply:
Oh wait I see.. this actually does a web request to strapped. ok hold on

Reply:
[update](https://github.com/CodeForBoulder/upswyng/pull/70/commits/980dd2b3a2895b25d07c59fd7d095de19f8805ca): removed `all.json` and updated gitignore